daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -9,7 +9,7 @@ import com.daml.lf.data.Ref._
 import com.daml.lf.language.Ast._
 import scala.annotation.tailrec
 
-private[lf] class PackageInterface(signatures: PartialFunction[PackageId, PackageSignature]) {
+private[lf] class PackageInterface(val signatures: PartialFunction[PackageId, PackageSignature]) {
```

Comment:
Using the signatures directly is convenient, but might be able to switch to lookupPackage, now that i'm not directly sequencing the PartialFunctions